usaspending_api/common/serializers.py

```diff
@@ -19,56 +19,64 @@ def __init__(self, *args, **kwargs):
             for field in children.keys():
                 child_args = {
                     **children[field].get("kwargs", {}),
-                    "context": {**self.context, "child": True},  # The child tag should be removed when child field limiting is implemented
-                }
+                    "context": {**self.context, "child": True}
+                }  # The child tag should be removed when child field limiting is implemented
                 self.fields[field] = children[field]["class"](**child_args)
         except AttributeError:
             # We don't have any nested serializers
             pass
 
         request = self.context.get('request')
-        if request and not self.context.get("child", False):  # The child check should be removed when child field limiting is implemented
-            # workaround to use this serializer with both GET and POST
+        if request:
```

Comment:
@afrasier Thanks for flagging! I removed the child check from this line to ensure that we display all fields on all child objects when the request is for a single item.

The child check is now being done on [line 44](https://github.com/fedspendingtransparency/usaspending-api/blob/support-single-item-verbose/usaspending_api/common/serializers.py#L44) and [line 53](url). The intent was to make sure that we don't mess with the serializer's field lists if we're in a nested serializer.

When testing, I got the same results in dev as in the branch. Can you clarify that what I'm describing is actually the issue you're raising? Would like to make sure this PR doesn't re-introduce an old issue.